CONTRIBUTING.md

```diff
@@ -66,6 +66,18 @@ If you would like to add new capabilities to the product as described in **B)**,
 
     Once the issue is created on GitHub, it still needs to be approved. Once approved, the rest of the process is identical to the process described in **A)**.
 
+## Get To Know System App
```

Comment:
I'm not a native speaker, but without the article, it sounds weird to me.
```suggestion
## Get To Know The System App
```



Do we need to capitalize every word?
```suggestion
## Get to Know the System App
```